.claude/skills/worktree/SKILL.md

```
@@ -0,0 +1,96 @@
+   ```
+
+4. **Copy config files**
+   - **unstract repo:** Run `"$SOURCE_REPO/.claude/skills/worktree/worktree-setup.sh" "$WORKTREE_PATH" "$SOURCE_REPO"`
+   - **Other repos:** Ask user which config files to copy, then copy them
+
+5. **Detect services from request** (unstract only)
+
+   Match keywords in user's request to services:
+
+   | Keywords | Services |
+   |----------|----------|
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe is missing space to the right for style "compact"

(MD060, table-column-style)